core/modules/system/system.module

```diff
@@ -837,16 +837,16 @@ function system_menu() {
     'access arguments' => array('access administration pages'),
     'file' => 'system.admin.inc',
   );
-  $items['admin/config/search/clean-urls'] = array(
-    'title' => 'Clean URLs',
-    'description' => 'Enable or disable clean URLs for your site.',
+  $items['admin/config/search/urls'] = array(
+    'title' => 'URL handling',
+    'description' => 'Handling URL-related configuration of your site.',
     'page callback' => 'backdrop_get_form',
-    'page arguments' => array('system_clean_url_settings'),
+    'page arguments' => array('system_urls_settings'),
     'access arguments' => array('administer site configuration'),
     'file' => 'system.admin.inc',
     'weight' => 5,
   );
```

Comment:
I suppose that would work, we could skip the extra function entirely though and just us backdrop_goto as the page callback.

```php
  // @deprecated: Remove backwards-compatibility redirect in 2.0.
  $items['admin/config/search/clean-urls'] = array(
    'page callback' => 'backdrop_goto',
    'page arguments' => array('admin/config/search/urls'),
    'access arguments' => array('access administration pages'),
    'type' => MENU_CALLBACK,
    'file' => 'system.admin.inc',
  );
```